ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -194,7 +178,13 @@ objects:
             fi
             if [[ -z "${ROUTER_NAMESPACE}" ]]; then
               # check multiple namespaces while we are transitioning to the new locations
-              if oc get ds/router-default -n openshift-ingress 2>/dev/null; then
+              if oc get deploy/router-default -n openshift-ingress 2>/dev/null; then
+                ROUTER_NAMESPACE=openshift-ingress
+                ROUTER_DEPLOYMENT="deploy/router-default"
+              elif oc get deploy/router -n tectonic-ingress 2>/dev/null; then
```

Comment:
We don't need a `tectonic-*` check, do we?

Reply:
Do we need it in the other  templates? I see it in every ci-operator/templates/openshift/installer/cluster-launch-installer-* template.  We can remove them all in a separate PR?

Reply:
Yeah, I'm fine punting then.